util/topsql/reporter/reporter.go

```diff
@@ -346,6 +452,12 @@ func (tsr *RemoteTopSQLReporter) takeDataAndSendToReportChan(collectedDataPtr *m
 	for _, evict := range evicted {
 		normalizedSQLMap.LoadAndDelete(string(evict.SQLDigest))
 		normalizedPlanMap.LoadAndDelete(string(evict.PlanDigest))
+		others = addEvictedDataPoints(others, evict)
+	}
+
+	// append others which summarize all evicted item's cpu-time.
+	if others != nil && others.CPUTimeMsTotal > 0 {
```

Comment:
```suggestion
	if others.CPUTimeMsTotal > 0 {
```
When will `others == nil`? AFAIK, `addEvictedDataPoints` will always returns a valid `*dataPoints`.

Reply:
No, when no evict happens, the `others` will be nil.